CI is red

There's a missing package `eslint-plugin-ckeditor5-rules` in the NPM repository and because of that lerna fails during the boostrap - https://travis-ci.org/ckeditor/ckeditor5-dev/builds/461202972#L466 

cc @pomek @Reinmar @jodator

Note that becasue of that you also can't use `lerna` locally:

```
npx lerna bootstrap
lerna notice cli v3.5.1
lerna info versioning independent
lerna info Bootstrapping 8 packages
lerna info Installing external dependencies
lerna info hoist Installing hoisted dependencies into root
lerna ERR! npm install exited 1 in 'ckeditor5-dev'
lerna ERR! npm install stderr:
npm WARN deprecated circular-json@0.5.9: CircularJSON is in maintenance only, flatted is its successor.
npm WARN deprecated text-encoding@0.6.4: no longer maintained
npm ERR! code E404
npm ERR! 404 Not Found: eslint-plugin-ckeditor5-rules@^1.0.0
```

--- a/packages/eslint-config-ckeditor5/package.json
+++ b/packages/eslint-config-ckeditor5/package.json
@@ -12,7 +12,7 @@
   ],
   "main": ".eslintrc.js",
   "dependencies": {
-    "eslint-plugin-ckeditor5-rules": "^1.0.0"
+    "eslint-plugin-ckeditor5-rules": "^0.0.1"
   },
   "peerDependencies": {
     "eslint": ">=3.0.0"

